ivy_tests/test_ivy/test_frontends/test_paddle/test_nn/test_functional/test_loss.py

```diff
@@ -390,3 +390,48 @@ def test_paddle_margin_ranking_loss(
         margin=margin,
         reduction=reduction,
     )
+
+@handle_frontend_test(
+    fn_tree="paddle.nn.functional.multilabel_soft_margin_loss",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+        num_arrays=2,
+        exclude_min=True,
```

Comment:
I believe that we can add this if we set the bounds of both `max` and `min` values of arrays to be generated